nnpdf_data/nnpdf_data/commondata/ATLAS_Z0_7TEV_36PB/metadata.yaml

```
@@ -42,35 +42,36 @@ implemented_observables:
       - $Z$
       - $Z$
       - $Z$
-    plot_x: k1
+    plot_x: abs_eta
   kinematic_coverage:
-  - k1
-  - k2
-  - k3
+  - abs_eta
+  - m_Z2
+  - sqrts
   kinematics:
     variables:
-      k1:
-        description: Variable k1
-        label: k1
+      abs_eta:
+        description: Absolute value of the rapidity of the Z boson
+        label: abs_eta
```

```suggestion
        label: "$y$"
```

Since it is rapidity and not pseudorapidity better to use `y`. At some point we were using eta/y to automatize a bit things (you can change also the `xlabel` to `$\left\| y\right|$` for instance) and so, some datasets that should be `Y` ended up being `ETA`. At some point I guess we'll update them.

---

I have just updated this